### ISSUE TITLE
setDevMode() not available

### ISSUE THREAD

**Issue by francoisduchemin** [admob-plus/admob-plus#19]
I'm trying `admob-plus` with ionic and I don't see any `setDevMode()` method in the `AdMob` object.

Maybe there is an issue with the typings.
`index.d.ts` shows the following for `AdMob`:
```
export declare class AdMob extends IonicNativePlugin {
    banner: Banner;
    interstitial: Interstitial;
    rewardVideo: RewardVideo;
    setAppMuted(value: boolean): Promise<any>;
    setAppVolume(value: number): Promise<any>;
    on(event: string): Observable<any>;
}
```

**Comment by ratson** [comment]
Added in v0.6.0

**Comment by francoisduchemin** [comment]
Sorry to reply to a closed issue, but where is v0.6.0?
If I'm not mistaken, the current version is 5.5.1.
https://github.com/ionic-team/ionic-native/blob/master/src/%40ionic-native/plugins/admob-plus/index.ts
